news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,171 @@
+And sometimes, the PrestaShop developers Team might ask us to test a PR as it is required for other topics.
+
+### How long does it take to test a Pull Request ?
+
+It highly depends on the complexity of the PR ! A small fix, with a clear issue and a clear “How to Test” can take only fifteen minutes to test, while a huge Migration PR can take several days.
+
+If a PR is really complex, or if the feature being implemented is large, there might be a second or even a third test by another QA specialist to validate it fully.
+
+### Why does my Pull Request have both the “Waiting for author” and “Waiting for QA” labels  ?
+
+If the tester in charge has a question on your PR (for example ask for a more explicit “How to test”), he will add the “Waiting for author” label, but without removing the “Waiting for QA” ones (in order to be able to check easily what are the PR waiting for a response from the author). So this means more information is needed, check the last comments on the PR !
+
+# II. Test of bug fix Pull Request
+
+PR can be classified into 3 main categories (with a few special cases) :
+- Bug fix
+- Improvement
+- Migration
+
+I will first focus on the bug fix Pull Request, since it’s the most common type there is, then detail the difference for improvement and migration PR.
+
+### Requirements
+
+Several things are required in order to have a PR being tested on our Github repository.
+First of all, the PR template must be followed, and all questions should be answered:
+- An issue should be linked to the PR. Each bug has to be [reproduced and qualified](https://build.prestashop.com/news/how-bug-reports-are-handled/) before being fixed. This helps the team to answer several questions : Is this really a bug ? What is the severity (see [Severity Classification](https://build.prestashop.com/news/severity-classification/)) ? What behavior is expected ? On which branch should it be fixed ?
+Of course, you’re welcome to fix issues submitted by another contributor (as long as the issue is linked to the PR) : This is the magic of open source !
+- The “How to test” field must be filled. It must contain the steps to check the bug is indeed fixed. If possible, it should also contain some information about the possible side-effects of the PR, in order to help check if the PR does not introduce regressions. Without a clear “how to test”, it’s impossible for us to know what to check on the PR.
+The most important thing about this part is that it must be a functional “How to test” :  PR are tested by functional manual testers, not technical ones. If your PR needs to make some technical changes in order to be tested (use a certain hook in the code for example): either make a test module allowing us to test it, or your PR will be tested by the maintainers team.
+
+![alt text](https://github.com/PrestaShop/prestashop.github.io/assets/images/2020/12/clear_howtotest.png "Clear How to Test in a PR")
```

Comment:
```suggestion
![Clear How to Test in a PR](https://github.com/PrestaShop/prestashop.github.io/assets/images/2020/12/clear_howtotest.png "Clear How to Test in a PR")
```
Alt text is what is shown for people who cannot display the image 😉  (bad network for example)